Source/WebCore/css/SelectorPseudoClassAndCompatibilityElementMap.in

```diff
@@ -85,6 +85,7 @@ fullscreen
 -webkit-full-screen-ancestor
 -webkit-full-screen-document
 -webkit-full-screen-controls-hidden
+-webkit-in-window-full-screen
```

Comment:
Please prefix with `-internal-` instead of `-webkit-` and make this UA sheet only, see: https://searchfox.org/wubkat/rev/3943142d2ef1777fc8977e6158eb10e053c3f8ca/Source/WebCore/css/parser/CSSSelectorParser.cpp#783-784

Reply:
Got it! Thanks